src/stylesheets/core/_variables.scss

```diff
@@ -1007,44 +1007,6 @@ then
 ----------------------------------------
 */
 
-/*
-----------------------------------------
-Typography
-----------------------------------------
-TODO: Rethink naming and consolidate in
-settings
-----------------------------------------
-*/
-
-// Removing the !default from $em-base so we are not inheriting that value from Bourbon.
-
-$em-base:             $theme-font-size-root;
-$base-font-size:      1.7rem !default;
-$small-font-size:     1.4rem !default;
-$lead-font-size:      2rem !default;
-$title-font-size:     5.2rem !default;
-$h1-font-size:        4rem !default;
-$h2-font-size:        3rem !default;
-$h3-font-size:        2rem !default;
-$h4-font-size:        1.7rem !default;
-$h5-font-size:        1.5rem !default;
-$h6-font-size:        1.3rem !default;
-$base-line-height:    1.5 !default;
-$heading-line-height: 1.3 !default;
-$lead-line-height:    1.7 !default;
-
-/*
-----------------------------------------
-Font weights
-----------------------------------------
-TODO: Consolidate in settings
-----------------------------------------
-*/
-
-// TODO these should be called font-weight-${}
-$font-normal:         $theme-font-weight-normal;
-$font-bold:           $theme-font-weight-bold;
-
 /*
 ----------------------------------------
 Colors
```

Comment:
Encountered some Sass errors when trying to move colors to settings file, left it for now, but it should be moved.

Reply:
What of these can be removed?